modules/nw-multus-host-device-object.adoc

```diff
@@ -54,7 +55,7 @@ plug-in:
   "hwaddr": "<hwaddr>", <3>
   "kernelpath": "<kernelpath>", <4>
   "pciBusID": "<pciBusID>", <5>
-    "ipam": { <6>
+  "ipam": { <6>
```

Comment:
Might be unavoidable, but the `...` below is rendered in red, as in there's an error in the syntax. I see it was like that before, but I also am pretty sure I've seen `...` used in YAML before without being rendered red. Not sure if it's because it's in `{}`, but maybe take a peek (now or later) if it can be resolved?

Reply:
Yoi... it's JSON not YAML, what was I thinking ;) Unsure about `...` in JSON then.